sw/host/rom_ext_image_tools/src/image/image.rs

```diff
@@ -0,0 +1,166 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::path::{Path, PathBuf};
+use std::fs::File;
+use std::io::{Read, Write};
+use std::ffi::OsString;
+
+use crate::image::manifest;
+use hex::FromHex;
+use serde_derive::Deserialize;
+use serde_hjson::Value;
```

Comment:
I personally separate imports from different crates into separate stanzas (with the current `crate` at the end) but that's just my own convention.